tile/TileSeparateCrossbarRTL.py

```diff
@@ -186,23 +186,44 @@ def update_opt():
       s.routing_crossbar.recv_opt.msg @= s.ctrl_mem.send_ctrl.msg
       s.fu_crossbar.recv_opt.msg @= s.ctrl_mem.send_ctrl.msg
 
-      s.element.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-      s.routing_crossbar.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-      s.fu_crossbar.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-
-      s.ctrl_mem.send_ctrl.rdy @= s.element.recv_opt.rdy & \
-                                  s.routing_crossbar.recv_opt.rdy & \
-                                  s.fu_crossbar.recv_opt.rdy
-
+      s.element.recv_opt.val @= s.ctrl_mem.send_ctrl.val & ~s.element_done

```

Comment:
@yo96 I saw python gave warning about `~` gonna be deprecated. Here we can use either `~` or `!` for now, and they have higher priority than `&`, right?

Reply:
Right. What does the warning message say? I've never encountered such warnings before and I couldn't find any relevant information.